File: spec/code_samples/Php_http2/list@{list_id}/delete.php
<?php

$client = new http\Client;
$request = new http\Client\Request;

$request->setRequestUrl('https://analyzemail.com/api/v1/list/%7Blist_id%7D');
$request->setRequestMethod('DELETE');
$request->setQuery(new http\QueryString(array(
  'username' => 'SOME_STRING_VALUE'
)));

$request->setHeaders(array(
  'content-type' => 'application/json'
));

$client->enqueue($request)->send();
$response = $client->getResponse();

echo $response->getBody();

File: spec/code_samples/Python_requests/list@{list_id}/delete.py
import requests

url = "https://analyzemail.com/api/v1/list/%7Blist_id%7D"

querystring = {"username":"SOME_STRING_VALUE"}

headers = {'content-type': 'application/json'}

response = requests.request("DELETE", url, headers=headers, params=querystring)

print(response.text)